routes/voice.js

```diff
@@ -0,0 +1,131 @@
+const debug = require('debug')('correlations-game:routes:voice');
+const express = require('express');
+const router = express.Router();
+
+const games = require('../bin/lib/game');
+let expectedAnswers = [];
+const not_understood_limit = 3;
+let not_understood_count = 0;
```

Comment:
These variables are global. Different games on different devices would be accessing/affecting the same variables.